.github/workflows/publish-benchmark.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trigger Site Rebuild on a CRON Schedule
+
+on:
+  schedule:
+    # Runs "at minute 55 past every hour" (see https://crontab.guru)
+    - cron: '0 2 * * *'
+jobs:
+  benchmark:
+    name: Run Benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Do you need to specify a branch or will it just checkout the default branch when using a schedule?